api/custom_auth/oauth/serializers.py

```diff
@@ -25,6 +26,7 @@ class Meta:
 
     def create(self, validated_data):
         user = self._get_user()
+        update_last_login(None, user)
```

Comment:
I quite like this approach but I'm not _certain_ we should be using the `update_last_login` function directly like this - it looks like it's a function used by django on the `user_logged_in` signal. Maybe we should in fact just trigger the signal? Something like this: 

```python
user_logged_in.send(get_user_model(), instance=user)
```

That way, we can use that signal in the future if we wanted as well.